src/Management/src/Endpoint/ManagementHostBuilderExtensions.cs

```diff
@@ -329,6 +330,21 @@ public static IHostBuilder AddTraceActuator(this IHostBuilder hostBuilder)
         return AddTraceActuator(hostBuilder, MediaTypeVersion.V2);
     }
 
+    /// <summary>
+    /// Adds the Services actuator to the application.
+    /// </summary>
+    /// <param name="hostBuilder">
+    /// Your HostBuilder.
+    /// </param>
+    public static IHostBuilder AddServicesActuator(this IHostBuilder hostBuilder)
```

Comment:
On second thought, we shouldn't be offering this on host builders at all. It solely needs the `IServiceCollection`, so there's no point in cluttering the API with this.

It's easy enough to write:
```c#
builder.Services.AddServicesActuator();
```
instead of:
```c#
builder.AddServicesActuator();
```

This also means the logic in `ManagementHostBuilderExtensions.ActivateActuatorEndpoints` needs to be moved into `EndpointServiceCollectionExtensions.AddServicesActuator`.

Does this make sense?